src/charm.py

```diff
@@ -269,21 +273,22 @@ def _get_srsenb_command(self) -> str:
                 f'--enb_files.rr_config={CONFIG_PATHS["rr"]}',
                 f'--enb_files.sib_config={CONFIG_PATHS["sib"]}',
                 f'--enb_files.drb_config={CONFIG_PATHS["drb"]}',
+                CONFIG_PATHS["enb"],
```

Comment:
So it was useful in the end?

Reply:
seems like it. the command looks for the config files in `/etc/srslte`.
unless the config files are copied there, we need to specify their path in the command.